src/main/java/org/broadinstitute/hellbender/tools/exome/CalculateCoverageComponentsInstance.java

```diff
@@ -0,0 +1,39 @@
+package org.broadinstitute.hellbender.tools.exome;
+
+import org.broadinstitute.hellbender.Main;
+import org.broadinstitute.hellbender.cmdline.StandardArgumentDefinitions;
+import org.broadinstitute.hellbender.utils.SparkToggleCommandLineProgram;
+
+import java.io.File;
+
+/**
+ * {@link CalculateCoverageComponents} running class.
+ *
+ * @author Valentin Ruano-Rubio &lt;valentin@broadinstitute.org&gt;
+ */
+final class CalculateCoverageComponentsInstance extends Main {
```

Comment:
it seems awkward to extend Main in this way.  will this only be used in the test code?  if so, perhaps make it a nested class in the test class?

not really sure a better way to get around this, but I think that perhaps the real problem is that there is no SparkToggleCommandLineProgramTest class.